packaging/centos7/cloud-management.service

```diff
@@ -24,6 +24,7 @@ Description=CloudStack Management Server
 After=syslog.target network.target
 
 [Service]
+UMask=0022
```

Comment:
@rafaelweingartner No, the value is not used as it is now. UMask=0022 is default and hence setting it has no purpose afaik.